packages/shared/src/modules/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -6,6 +6,7 @@ const UserValidationMessage = {
 	FIRST_NAME_REQUIRED: "First name is required.",
 	LAST_NAME_REQUIRED: "Last name is required.",
 	PASSWORD_MINIMUM_LENGTH: `Password must be at least ${String(UserValidationRule.PASSWORD_MINIMUM_LENGTH)} characters long.`,
+	PASSWORDS_DO_NOT_MATCH: "Passwords do not match",
```

Comment:
I see you added this message, but I don't see it's being applied to the validation schema. Also, let's end validation messages with a dot to have a fully correct sentence (the latest best practices I know)

```suggestion
	PASSWORDS_DO_NOT_MATCH: "Passwords do not match.",
```